style.css

```diff
@@ -0,0 +1,71 @@
+html,
+body {
+  height: 100%;
+  background: url("images/header_bg.png");
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-position: center center;
+  background-size: 110%;
+  box-sizing: border-box;
+}
+
+nav {
+  display: flex;
+  justify-content: space-around;
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+  height: 90%;
+  justify-content: flex-end;
+  padding-left: 3%;
+  padding-right: 5%;
+  margin-top: 215px;
+}
+
```

Comment:
- [ ] Kindly give your design the right `margin-top` so it looks as close as possible to your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/191930392-0288b30c-abce-4ea6-868f-7d362191c6a6.png)  |  ![image](https://user-images.githubusercontent.com/74814780/191930415-77e3fe9e-ccb5-42b4-bef3-543e0337db42.png)